crates/cxx-qt-gen/src/generator/rust/inherit.rs

```diff
@@ -45,14 +45,12 @@ pub fn generate(
             if method.safe {
                 std::mem::swap(&mut unsafe_call, &mut unsafe_block);
             }
-
-            let attrs = &method.method.attrs;
             let doc_comments = &method.docs;
 
             syn::parse2(quote_spanned! {
                 method.method.span() =>
                 #unsafe_block extern "C++" {
-                    #(#attrs)*
+                    // TODO: should namespace be added here?
```

Comment:
good question, it seems to work at the moment or maybe a test case doesn't try this out yet. Would be good to check that CXX generates the right thing when the `cxx::bridge` does not have a namespace, as this extern block doesn't have a namespace either, but the self QObject is in a namespace.